SrProj/Static/controllers/appController.js

```diff
@@ -66,6 +66,17 @@ $(function() {
         vm: loginVM
       },
 
+        PatronCheckIn: {
```

Comment:
Your tabbing is off, but I'm pretty sure that's VS's fault. I can fix later if VS gives you shit about it.